docs/cli/getting-started.md

```diff
@@ -9,59 +9,114 @@ A simple set of instructions to set up and use the Tanzu CLI.
 `linux-amd64`,`windows-amd64`, and `darwin-amd64` are the OS-ARCHITECTURE
 combinations we support now.
 
-If you want to install the latest release of the Tanzu CLI, you can run the below commands:
+#### macOS/Linux
 
-#### MacOS/Linux
+- Download the latest [release](https://github.com/vmware-tanzu/tanzu-framework/releases/latest)
 
-```shell
-export ARCH=$(case $(uname -m) in x86_64) echo -n amd64 ;; aarch64) echo -n arm64 ;; esac)
-export OS=$(uname | awk '{print tolower($0)}')
+- Extract the downloaded tar file
 
-curl -o tanzu https://storage.googleapis.com/tanzu-cli/artifacts/core/latest/tanzu-core-${OS}_${ARCH} && \
-    mv tanzu /usr/local/bin/tanzu && \
-    chmod +x /usr/local/bin/tanzu
-```
+  ```sh
+  mkdir tanzu && tar -xvf tanzu-framework-darwin-amd64.tar -C tanzu
+  ```
+
+- Install the `tanzu` CLI
+
+  ```sh
+  sudo install tanzu/cli/core/v0.5.0/tanzu-core-darwin_amd64 /usr/local/bin/tanzu
```

Comment:
this references mac tarball in the mac/linux section

Reply:
updated